client/js/lounge.js

```diff
@@ -729,6 +729,7 @@ $(function() {
 		var self = $(this);
 		viewport.toggleClass(self.attr("class"));
 		e.stopPropagation();
+		chat.find(".chan.active .chat").trigger("msg.sticky");
```

Comment:
I was expecting that if I was not stick'ed to bottom and clicked on the user list button this would force-stick me but it doesn't.
It works as expected, i.e. sticks you to bottom only if you were before clicking, but I can't figure out why. Do you happen to know what magic is this, @xPaw? If I read this correctly, any click should stick you to bottom regardless of scroll position before clicking.